src/promise/pTo.test.ts

```diff
@@ -0,0 +1,30 @@
+import { pTo } from './pTo'
+
+const FailingPromise = new Promise((resolve, reject) => {
+  reject(new Error('FAILED'))
+})
+
+const ResolvingPromise = new Promise<boolean>(resolve => {
+  resolve(true)
+})
+
+test('TO should catch async errors', async () => {
+  const [error] = await pTo(FailingPromise)
+
+  expect(!!error).toBe(true)
+  expect(error instanceof Error).toBe(true)
```

Comment:
Not important, but I would write e.g `expect(error).toThrowErrorMatchingInlineSnapshot(...)`, which would also test that the error thrown is not modified in any way.